packages/playground/cli/src/blueprints-v1/worker-thread-v1.ts

```diff
@@ -202,7 +222,72 @@ export class PlaygroundCliBlueprintV1Worker extends PHPWorker {
 	}
 
 	async bootAsSecondaryWorker(args: WorkerBootOptions) {
-		return this.bootAsPrimaryWorker(args);
+		await this.bootRequestHandler(args);
+	}
+
+	async bootRequestHandler({
+		siteUrl,
+		allow,
+		php,
+		firstProcessId,
+		processIdSpaceLength,
+		trace,
+		nativeInternalDirPath,
+		mountsBeforeWpInstall,
+		mountsAfterWpInstall,
+	}: WorkerBootRequestHandlerOptions) {
+		if (this.booted) {
+			throw new Error('Playground already booted');
+		}
+		this.booted = true;
+
+		let nextProcessId = firstProcessId;
+		const lastProcessId = firstProcessId + processIdSpaceLength - 1;
+
+		try {
+			const requestHandler = await bootRequestHandler({
+				siteUrl,
+				createPhpRuntime: async () => {
+					const processId = nextProcessId;
+
+					if (nextProcessId < lastProcessId) {
+						nextProcessId++;
+					} else {
+						// We've reached the end of the process ID space. Start over.
+						nextProcessId = firstProcessId;
+					}
+
+					return await loadNodeRuntime(php!, {
+						emscriptenOptions: {
+							fileLockManager: this.fileLockManager!,
+							processId,
+							trace: trace ? tracePhpWasm : undefined,
+							ENV: {
+								DOCROOT: '/wordpress',
+							},
+							phpWasmInitOptions: { nativeInternalDirPath },
+						},
+						followSymlinks: allow?.includes('follow-symlinks'),
+					});
+				},
+				async onPHPInstanceCreated(php) {
+					await mountResources(php, mountsBeforeWpInstall);
+					await mountResources(php, mountsAfterWpInstall);
+				},
+				sapiName: 'cli',
```

Comment:
We can probably remove this in a follow-up pr since we have a proper `php.cli()` implementation working across all runtimes.

Reply:
> Yes! We used to pretend we're running a CLI SAPI when we really have been dispatching a HTTP request via the WASM (old embed) SAPI. We no longer do that.

Got it! I'll remove the declarations in a follow-up PR.